### PR TITLE
[server] remove prebuilt time limit

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description
Removes the unnecessary check of prebuild times. The query is logged to take around 7 secs.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **svenefftinge** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/gitpod-db/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-db/OWNERS)**
- **[components/licensor/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/licensor/OWNERS)**
- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/gitpod-db/src/typeorm/migration/1592203031938-Baseline.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ export class Baseline1592203031938 implements MigrationInterface {
         {
             const exists = (await queryRunner.query(`SELECT COUNT(1) AS cnt FROM d_b_user WHERE id = 'builtin-user-workspace-probe-0000000'`))[0].cnt == 1;
             if (!exists) {
-                await queryRunner.query(`INSERT IGNORE INTO d_b_user (id, creationDate, avatarUrl, name, fullName) VALUES ('builtin-user-workspace-probe-0000000', '${new Date().toISOString()}', '', '${BUILTIN_WORKSPACE_PROBE_USER_NAME}', '')`)
```
Comment (geropl) [inline components/gitpod-db/src/typeorm/migration/1592203031938-Baseline.ts:84]: Is this guaranteed, or do we have environments where there builtin user has another ID (for whatever reason)?
I know it seems obvious from the code. I just assume there was some motivation to use the user name in the first place, and want to be sure we don't miss sth here. :slightly_smiling_face:

Comment (geropl) [reply components/gitpod-db/src/typeorm/migration/1592203031938-Baseline.ts:84]: @csweichel

File: components/gitpod-db/src/typeorm/workspace-db-impl.ts [hunk 2]
```diff
@@ -574,25 +572,6 @@ export abstract class AbstractTypeORMWorkspaceDBImpl implements WorkspaceDB {
             .getOne();
     }
 
-    public async getTotalPrebuildUseSeconds(forDays: number): Promise<number | undefined> {
-        const manager = await this.getManager();
-        const res = await manager.query(`
-            SELECT SUM(COALESCE(STR_TO_DATE(wsi.stoppedTime, "%Y-%m-%dT%H:%i:%s.%fZ") - STR_TO_DATE(wsi.startedTime, "%Y-%m-%dT%H:%i:%s.%fZ"))) AS tps FROM d_b_workspace_instance wsi
```
Comment (geropl) [inline components/gitpod-db/src/typeorm/workspace-db-impl.ts:580]: If it was just for the query time we could optimize this for sure. But I assume it's a product-decision to remove this constraint and move to sth else based on growth for Enterprise (e.g., "free for up to 10 seats, pay for more").

File: components/gitpod-db/src/typeorm/migration/1592203031938-Baseline.ts (hunk 1 above)
Comment (svenefftinge) [reply components/gitpod-db/src/typeorm/migration/1592203031938-Baseline.ts:84]: We have both in staging and prod one user with that id and multiple users (3 in staging, 4 in prod) with that name.

**Comment by geropl** [comment]
/lgtm

Code looks good, preview works.

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 0b765f4523d11df4f73c9b09b7e74ebd8a1caab3</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5679#issuecomment-918962524" title="LGTM">geropl</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/gitpod-db/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-db/OWNERS)~~ [geropl]
- ~~[components/licensor/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/licensor/OWNERS)~~ [geropl]
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [geropl]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5679#issuecomment-918962524" title="LGTM">geropl</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5679#issuecomment-919007882" title="Approved">JanKoehnlein</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5679#issuecomment-919007882" title="Approved">JanKoehnlein</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/gitpod-db/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-db/OWNERS)~~ [JanKoehnlein,geropl]
- ~~[components/licensor/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/licensor/OWNERS)~~ [JanKoehnlein,geropl]
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [JanKoehnlein,geropl]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**